### PR TITLE
Add chart message support for ACLK new architecture

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
Add initial support for chart/dimension update messages

##### Component Name
aclk, database

##### Test Plan
- The code is not active yet (it is tested with the new architecture in a separate branch)

### CODE REVIEW

File: database/sqlite/sqlite_functions.h [hunk 1]
```diff
@@ -63,6 +63,7 @@ extern int find_uuid_type(uuid_t *uuid);
 extern void sql_rrdset2json(RRDHOST *host, BUFFER *wb);
 
 extern RRDHOST *sql_create_host_by_uuid(char *guid);
+extern int prepare_statement(sqlite3 *database, char *query, sqlite3_stmt **statement);
```
Comment (underhood) [inline database/sqlite/sqlite_functions.h:66]: in sqlite_functions.c you have removed static from 2 functions:
- execute_insert
- prepare_statement
but in sqlite_functions.h you added only
- prepare_statement

Comment (stelfrag) [reply database/sqlite/sqlite_functions.h:66]: I will fix it  (now it exists in sqlite_aclk_chart.h)

File: database/sqlite/sqlite_aclk.h [hunk 2]
```diff
@@ -157,7 +154,7 @@ struct aclk_database_cmd {
     struct aclk_completion *completion;
 };
 
-#define ACLK_DATABASE_CMD_Q_MAX_SIZE (2048)
+#define ACLK_DATABASE_CMD_Q_MAX_SIZE (16384)
```
Comment (underhood) [inline database/sqlite/sqlite_aclk.h:157]: do we have any reasoning behind this? Was this increased to support higher number of children?

**Review by underhood — APPROVED** [review]

File: database/sqlite/sqlite_aclk.h (hunk 2 above)
Comment (stelfrag) [reply database/sqlite/sqlite_aclk.h:157]: This will be further refined (prevents blocking during chart/dimension creation)

**Comment by stelfrag** [comment]
Rebased to resolve conflicts

**Review by underhood — APPROVED** [review]

**Review by MrZammler — APPROVED** [review]

**Pull request merged**